### PR TITLE
Return original mentionItem on value input

### LINKED ISSUE #15: onMentioned doesnt return the original object
Hey There,

When you select an item from the tribute list you only get a string value returned

```
onMentioned = new EventEmitter<string>();
```

now this does work when you are working with simple objects such as

```
{ key: 'baz', value: 'Baz' }
```

but in more advanced cases an object may look more like this

```
{
    name: 'John Smith',
    avatar: 'https://linktoavatar.com/picture.jpg',
    id: 32,
    email: 'johnsmith@gmail.com'
}
```

now say I select my user 'John Smith' - onMentioned only returns `@John Smith`

I suggest returning the string and the original object the `addEventListener()` function can be changed to the following

```
export class MentionObject {
   string: string,
   original: any
}

//...

@Output()
onMentioned = new EventEmitter<MentionObject>();

this.element.nativeElement.addEventListener('tribute-replaced', (event) => {
    const value = ['INPUT', 'TEXTAREA'].includes(
         this.element.nativeElement.tagName
    )
         ? this.element.nativeElement.value
          : this.element.nativeElement.innerHTML;

         this.onMentioned.emit({
             string: value,
             original: event.detail.item.original
         });

         if (this.control) {
             this.control.setValue(value);
         }
});
``` 

now the onMentioned will return 

```
{
    string: '@John Smith',
    original: {
        name: 'John Smith',
        avatar: 'https://linktoavatar.com/picture.jpg',
        id: 32,
        email: 'johnsmith@gmail.com'
    }
}
```

now a user could use any of the values from original item.

### PR DESCRIPTION
**Pull request by Alex61NN5**
closes #15

### CODE REVIEW

**Comment by agarbund** [comment]
Thank you! I'll check and merge it tonight. Please remove `yarn.lock`

**Comment by Alex61NN5** [comment]
@agarbund my bad, `yarn.lock` removed

File: src/app/app.component.ts [hunk 1]
```diff
@@ -25,6 +25,15 @@ interface TributeValue {
         <input [ngxTribute]="options" (onMentioned)="lastMention = $event">
         Value after mention: {{ lastMention }}
 
+        <h2>Retrieving the original item</h2>
+        <input [ngxTribut]="advancedOptions" (mentionItemSelected)="advancedMention = $event">
```
Comment (agarbund) [inline src/app/app.component.ts]:
```suggestion
        <input [ngxTribute]="advancedOptions" (mentionItemSelected)="advancedMention = $event">
```

**Comment by agarbund** [comment]
`Can't bind to 'ngxTribut' since it isn't a known property of 'input'`

Seems like a typo, can you please fix and verify if demo app is working correctly?

**Comment by Alex61NN5** [comment]
Hey, Sorry just fixed that issue and pushed!

**Pull request merged**